aten/src/ATen/core/op_registration/kernel_functor.h

```diff
@@ -176,11 +180,7 @@ namespace detail {
   private:
     template<size_t... indices>
     static void call_(std::tuple<OutputTypes...>&& output, Stack* stack, guts::index_sequence<indices...>) {
-      (void)(stack); // when sizeof...(indices) == 0, this argument would be unused and we have to silence the compiler warning.
-      // iterate over all outputs and push them
-      (void)std::initializer_list<int>{(
```

Comment:
why did we have this initializer_list in the first place?

Reply:
It is a trick to expand a parameter list into a series of statements. It was calling `torch::jit::push(stack, output)` on each output. But, actually, inside the implementation of `torch::jit::push`, there is already an `initializer_list` doing exactly the same, so instead of calling it on each output individually, we can just pass in `torch::jit::push(stack, output, ...)` and have that `initializer_list` take care of it.